src/redux/actionTypes.js

```diff
@@ -0,0 +1,6 @@
+export const actionTypes = {
+    getAllPokemon: 'getAllPokemon',
```

Comment:
Probably it would be better to create action types in the upper snake case instead of the camel case.

like getAllPokemon becomes GET_ALL_POKEMON.
The upper snake case defines a variable as a constant.